.gitignore

```diff
@@ -7,6 +7,7 @@
 _site/
 _build/
 cfgov/static_built/*
+cfgov/templates/wagtailadmin/js/*
```

Comment:
Dont know why we wouldnt need this we have a similar line:
cfgov/static_built/*
for the rest of the static built code. Maybe where the file outputs should be moved into cfgov/static_built?

Reply:
Oops, I forgot to follow-up on this in the course of finishing my review on Friday. I'll take another look today.

Reply:
Ahahhh, this is because of the gymnastics we did to put the JS file back where it was originally after processing. I've just played around with it and determined that that was unnecessary. Commit coming to clean this up.